src/main/java/com/telecom/antiphishing/exception/PhishingMessageException.java

```diff
@@ -0,0 +1,12 @@
+package com.telecom.antiphishing.exception;
+
+/***
+ * @author - Kiryl Karpuk
+ */
+public class PhishingMessageException extends RuntimeException {
+
+    public PhishingMessageException() {
+        super("Phishing detected");
+    }
+
+}
```

Comment:
Czy wykrycie phishing'u to jest błąd runtime? Czy wykrycie phishing'u to nie jest, jakby nie patrzeć, pożądany wynik?

Reply:
W tym przypadku exception ten nie jest w ogóle potrzebny. Ma to relację do [tego](https://github.com/CaulKeR/sms-security/pull/1#discussion_r1424432590) komentarza